pkg/controller/petset/pet.go

```diff
@@ -251,8 +251,8 @@ func (p *apiServerPetClient) SyncPVCs(pet *pcb) error {
 		if !exists {
 			if err := p.createPVC(&pet.pvcs[i]); err != nil {
 				errMsg += fmt.Sprintf("Failed to create %v: %v", pvc.Name, err)
+				p.event(pet.parent, "Create", fmt.Sprintf("pvc: %v", pvc.Name), err)
 			}
-			p.event(pet.parent, "Create", fmt.Sprintf("pvc: %v", pvc.Name), err)
```

Comment:
we want an event eitherway. Create succeeding is positive reinforcement, we're telling the user we created this volume for you. Create failing might mean eg: user is out of quota, no permissions to create volumes etc. 

The issue is that "err" is the getPVC error. 
A slightly deeper issue, is what should getPVC return if the volume doesn't exist? we already return exists = false, do we also need an error? we know it's NotFound, maybe just log it at V(4) and pass back exists=false, err=nil?